config/deploy.production.yml

```diff
@@ -0,0 +1,59 @@
+# Production deployment configuration
+service: skillrx-production
+
+# Name of the container image.
+image: andremuta/skillrx
+
+# Deploy to production server (update with your production server IP)
+servers:
+  web:
+    - 3.233.202.191
+
+proxy:
+  ssl: true
+  host: skillrx.org
```

Comment:
We probably want to account for folks typing in www.
ie : `host: www.skillrx.org, skillrx.org"`